lib/app.jsx

```diff
@@ -353,8 +356,8 @@ export const App = connect(
       // a note. The NoteEditor will notify via props when
       // it's time to sync via Simperium
       const { noteBucket } = this.props;
-
       noteBucket.update(note.id, updatedNote.data, {}, { sync: false });
+      this.syncNote(note.id);
```

Comment:
I think this new line defeats the purpose of the lines above it.

we want to call `syncNote()` when we select a revision, but not on every change otherwise we'd have a mass of granular edits in the revision.

Reply:
I have a local change that addresses this, still working o the PR. Not quite ready for review.